katas/langtonant/alfredo/module1.js

```diff
@@ -0,0 +1,140 @@
+export class class1 {
+    constructor() {
+        this.direction = 1 ;
+        this.matriz = new Array(10);
+        this.posi = new Array(2);
+        this.posi[0] = 5;
+        this.posi[1] = 5;
+        this.waymatriz = new Array(10);
+        
+        for (let i = 0; i < 10 ;i++) {
+            this.matriz[i] = new Array(10);
+            this.waymatriz[i] = new Array(10);
+        }
+        for (let i = 0; i < 10 ;i++) {
```

Comment:
There are some spaces that should be fixed with lint here, please make sure to run and fix all the issues detected by lint